### PR TITLE
Add pricing unit

### PR DESCRIPTION
**Pull request by RahulJha-mastek**
Story [AB#13749](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13749)
Task [AB#14022](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14022)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql [hunk 1]
```diff
@@ -83,7 +83,26 @@ VALUES
     ('02438278-ab93-4689-b123-7ac4e78f59fe', 'smsWebEmail_1000'    , 'responses per site'           , '1000 SMS/web/email responses per site'),
     ('67e1c174-8443-4883-b51f-43c297fa9c08', 'smsWebEmail_1500'    , 'responses per site'           , '1500 SMS/web/email responses per site'),
     ('1a36f980-76a4-4b3d-b36d-46afc6655124', 'dataMigration'       , 'data migrations'              , 'per data migration'),
-    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service');
+    ('8413d5e1-2995-4885-9ffa-ec961d82aa6d', 'sharedService'       , 'shared services'              , 'per shared service'),
+    ('f24afc45-971d-41fa-9beb-6f577e987c6d', 'patients_1-20k'      , 'patients'                     , 'per patient for 1-19,999 patients'),
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:87]: The hyphen-minus character should be an en dash, e.g. `'per patient for 1–19,999 patients'`.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql [hunk 2]
```diff
@@ -83,7 +83,26 @@ VALUES
...
+    ('0fb82859-aa6c-4db5-b513-c776c7278aca', 'patients_20-150k'    , 'patients'                     , 'per patient 20,000 - 149,999 patients'),
+    ('7b2c46d4-e5c1-4315-a05e-021e43d485ee', 'patients_150-250k'   , 'patients'                     , 'per patient 150,000 - 249,999 patients'),
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:89]: These should follow the same styling as above, i.e. with an en dash without spaces, for example:

```sql
'per patient 20,000–149,999 patients'
'per patient 150,000–249,999 patients'
```

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql [hunk 3]
```diff
@@ -83,7 +83,26 @@ VALUES
...
+    ('d37f63e0-0bb2-4703-9f86-f22057231a00', 'patients_250k'       , 'patients'                     , 'per patient 250,000 patients & above'),
+    ('950365df-456d-4c72-bc6f-74a0f7535728', 'patientRegistered'   , 'patients'                     , 'per registered patient'),
+    ('2c25c251-168d-4f14-a637-3d09fe7440b7', 'onlineConsultationGP', 'consultations'                , 'per online consultation non GP practices'),
+    ('edb80481-d99a-4d25-8d58-da6881b453f7', 'videoConsultationGP' , 'consultations'                , 'per video consultation non GP practices'),
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:93]: non-GP should be hyphenated, for example:

```sql
'per online consultation non-GP practices'
'per video consultation non-GP practices'
```

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql [hunk 4]
```diff
@@ -83,7 +83,26 @@ VALUES
...
+    ('a2441010-cfd0-4a0a-bbf7-e73e279b4298', 'referralEps_1-5k'    , 'referral episodes'            , 'per referral episode 1-5000 episodes'),
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:94]: As above, we should use an en dash here, e.g. `'per referral episode 1–5000 episodes'`.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql [hunk 5]
```diff
@@ -83,7 +83,26 @@ VALUES
...
+    ('c4bfbaf0-f311-47d2-b02c-7ee76b9f779f', 'referralEps_5-20k'   , 'referral episodes'            , 'per referral 5001 - 20,000 episodes'),
+    ('2f59e518-e211-44bb-b9b3-0fc518f4f46c', 'referralEps_20-50k'  , 'referral episodes'            , 'per referral 20,001 - 50,000 episodes'),
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:96]: These should follow the same styling as above, i.e. with an en dash without spaces, for example:

```sql
'per referral 5001–20,000 episodes',
'per referral 20,001–50,000 episodes'
```

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql [hunk 6]
```diff
@@ -83,7 +83,26 @@ VALUES
...
+    ('5c863dc7-b17f-43aa-997a-e5d1226525d8', 'referralEps_50k'     , 'referral episodes'            , 'per referral 50,0001 or more episodes'),
+    ('3d49fdd4-a949-40ab-adea-f541a6dbe034', 'trainingSession_360' , 'training sessions'            , 'per 4 x 90 min online training sessions'),
+    ('035c6c00-773b-47f7-af61-4eb60a1ce639', 'language'            , 'languages'                    , 'per Language'),
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:99]: units are usually lower case, so should be 'per language'.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql [hunk 7]
```diff
@@ -83,7 +83,26 @@ VALUES
...
+    ('2231d5e7-c9a8-4c77-ae35-ca9d5cc5d79a', 'solution'            , 'solutions'                    , 'per solution'),
+    ('1783eb6d-dbc0-4a00-bc4f-e9ce9331632c', 'pcn'                 , 'pcn'                          , 'per PCN'),
+    ('088655c4-83cb-40b6-a996-1f3eb9056fcd', 'call_1min'           , 'calls'                        , 'per 1 minute/part of a 1 min of call'),
+    ('4c791800-02a4-4e34-a6a5-575c08f37ad2', 'trainingSession_90'  , 'training sessions'            , 'per 90 min online training session'),
+    ('d88a3485-e2bc-4dc5-bb75-1ffb6df8a31d', 'smsSent'             , 'sms'                          , 'per sms sent'),
+    ('4a202b19-2cd0-4650-b4f7-b89e00762168', 'smsText'             , 'sms'                          , 'per SMS text');
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:105]: SMS should be consistently upper case, so the tiered names will need updating and `per SMS sent`.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
I'm guessing you probably just copied the pricing units – unfortunately, they often need cleaning up so I've recommended some small changes.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 7 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql:105]: we have specifically comment for this in Task [14022](https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_workitems/edit/14022)

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 1 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql]: Updated description to use hyphen-minus character.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 2 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql]: Updated description to use hyphen-minus character.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 3 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql]: Updated non-GP with hyphen

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 4 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql]: Updated description to use en dash character.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 5 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql]: Updated description to use en dash character.

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 6 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql]: updated unit to lowercase

File: src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql (hunk 7 above)
Comment (RahulJha-mastek) [reply src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql]: Updated description to `per SMS sent`

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**